src/components/plugins/MapQuestionnaire.js

```diff
@@ -40,18 +39,16 @@ class MapQuestionnaire extends BaseCommentForm {
             placeholder="Kommentoi ehdotustasi tässä."
           />
         </FormGroup>
-        {canSetNickname ? <h3><FormattedMessage id="nickname"/></h3> : null}
-        {canSetNickname ? (
-          <FormGroup>
-            <FormControl
-              type="text"
-              placeholder={this.props.intl.formatMessage({id: "anonymous"})}
-              value={this.state.nickname}
-              onChange={this.handleNicknameChange.bind(this)}
-              maxLength={32}
-            />
-          </FormGroup>
-        ) : null}
+        <h3><FormattedMessage id="nickname"/></h3>
```

Comment:
I have happy tidings: The `commentBox` is actually not needed in `MapQuestionnaire.js` anymore. It's a leftover from the old legacy map questionnaires.

The only place where `MapQuestionnaire` is rendered is in `PluginContent.js`, which automatically sets `displayCommentBox` prop to `false`. Therefore, we could just get rid of the prop `displayCommentBox` and all `commentBox` functionality here.